src/java.desktop/macosx/native/libawt_lwawt/java2d/metal/MTLTexurePool.m

```diff
@@ -202,6 +202,8 @@ - (MTLTexturePoolItem *)createItem:(id<MTLDevice>)dev
                                                                width:(NSUInteger) width
                                                               height:(NSUInteger) height
                                                            mipmapped:NO];
+    textureDescriptor.usage = MTLTextureUsageRenderTarget |
+        MTLTextureUsageShaderRead;
```

Comment:
In copyArea we use intermediate texture queried from TexturePool as render target. If we dont set any MTLTextureUsage, we are getting assertion errors when Metal API validation is enabled because by default we can only do texture shader read(https://developer.apple.com/documentation/metal/mtltexturedescriptor/1515783-usage?language=objc). To avoid these assertion errors in Metal API validation we need to set appropriate MTLTextureUsage parameters.